gef.py

```diff
@@ -8642,7 +8620,7 @@ def context_trace(self) -> None:
 
         # backward compat for gdb (gdb < 7.10)
         if not hasattr(gdb, "FrameDecorator"):
-            gdb.execute("backtrace {:d}".format(nb_backtrace))
+            gdb.execute(f"backtrace {nb_backtrace:d}")
             return
```

Comment:
It has nothing to do with your PR but since we now baseline gdb to be 8, can you remove this block?
```suggestion

```

Reply:
Added to project https://github.com/hugsy/gef/projects/11#card-75928305

Reply:
So should I still commit the suggested fix or now that it was added to the project to track, we do that in a future code change?

Reply:
Future one